jwst/pipeline/calwebb_spec2.py

```diff
@@ -252,6 +254,9 @@ def process_exposure_product(
         else:
             calibrated = self._process_common(calibrated)
 
+        # for MIR_MRS data call the residual fringe correction
```

Comment:
I wonder if it would be a little cleaner (in terms of code complexity) to move this call into the `_process_common` function, along with the `fringe` step. The setting of the `skip` param will still control whether or not it gets applied to data for different modes.

Reply:
David - currently there is no keyword for the FRINGEFREQ in the core schema. I need to make up one. Do you have a preference R_FRIFRQ ?

Reply:
R_FRIFRQ makes sense to me, but I defer to whatever you and Howard think most consistent with the other keywords.

Reply:
@drlaw1558 have you used the parameters 'ignore_region_min' or 'ignore_region_max' to not used wavelengths. I can not get these parameters to work anymore

Reply:
Can't say as I've ever tried using those before